modules/k8s/kubectl_options.go

```diff
@@ -2,18 +2,28 @@ package k8s
 
 import "testing"
 
-// Represents common options necessary to specify for all Kubectl calls
+// KubectlOptions represents common options necessary to specify for all Kubectl calls
 type KubectlOptions struct {
 	ContextName string
 	ConfigPath  string
 	Namespace   string
 	Env         map[string]string
 }
 
-func NewKubectlOptions(contextName string, configPath string) *KubectlOptions {
-	return &KubectlOptions{
-		ContextName: contextName,
-		ConfigPath:  configPath,
+// NewKubectlOptions will return a pointer to new instance of KubectlOptions with the configured options
+func NewKubectlOptions(opts ...string) *KubectlOptions {
+	switch {
+	case len(opts) = 3:
+		return &KubectlOptions{
+			ContextName: opts[0],
+			ConfigPath:  opts[1],
+			Namespace:   opts[2],
```

Comment:
While you are in here, can you also initialize the map for `Env`? Basically add `Env: map[string]string{}` to the struct creation. The reason is that by default it initializes to `nil`, and you can't start adding to `Env`, so by initializing, it makes the experience better.

Reply:
Will do

Reply:
added this in 12eaf30

Reply:
Thanks!